poetry/utils/helpers.py

```diff
@@ -77,3 +81,10 @@ def parse_requires(requires):  # type: (str) -> Union[list, None]
 
     if requires_dist:
         return requires_dist
+
+
+def load_dotenv_if_exists():
+    workdir_path = Path(os.getcwd())
+    env_file_path = workdir_path / ".env"
+    if env_file_path.is_file():
+        load_dotenv(env_file_path)
```

Comment:
Might be overthinking it but if we did something like

```
read_dotenv = dotenv.get_key(env_file_path, 'POETRY_READ_DOTENV')
if read_dotenv and read_dotenv.lower() not in ('false', 'n', 'no', 'off', '0'):
    load_dotenv(env_file_path)
```

This gives users a way to turn off reading the .env file on a per-directory basis even if we have a global setting that turns it on?